test/intern/mixins/Tree.js

```diff
@@ -409,6 +409,22 @@ define([
 					}, null, 'Modification of child should not throw error');
 				});
 			});
+
+			test.test('collapse items removed from store', function() {
+				return expand(0).then(function() {
+					assert.isTrue(grid.row(0).element.classList.contains('dgrid-row-expanded'),
+						'Should have expanded class');
+					var item = grid.collection.getSync(0);
+					grid.removeRow(grid.row(0));
```

Comment:
This kind of operation shouldn't ordinarily be performed directly for a store-based grid; I'd suggest just dropping lines 418-421 from this test unless you have a specific reason for keeping them.